<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OMERewriter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupQuickActions">
            <property name="title">
             <string>Quick Actions</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnLoadTiff">
               <property name="text">
                <string>Load TIFF</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnQuickSave">
               <property name="text">
                <string>Quick Save</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupTiffInterpretation">
            <property name="title">
             <string>TIFF Interpretation</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="interleavedChannelsLabel">
               <property name="text">
                <string>Interleaved Channels</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinCInterleaveCount"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="MicroscopeParamsWidget" name="imageMetaWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="viewDockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="ImageViewWidget" name="imageView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="navigationGroup">
       <property name="title">
        <string>Navigation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelZ">
          <property name="text">
           <string>Z:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="sliderZ">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TickPosition::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinBoxZ">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelT">
          <property name="text">
           <string>T:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="sliderT">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TickPosition::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBoxT">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelC">
          <property name="text">
           <string>C:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="sliderC">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TickPosition::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinBoxC">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageViewWidget</class>
   <extends>QWidget</extends>
   <header>imageviewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MicroscopeParamsWidget</class>
   <extends>QWidget</extends>
   <header>microscopeparamswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
